guide/designing-products/lightning-service-providers.md

```diff
@@ -0,0 +1,306 @@
+---
+layout: guide
+title: Lightning Service Providers
+description: Common user experience problems of the Lightning network that can be resolved by third-party services.
+nav_order: 10
+parent: Designing bitcoin products
+permalink: /guide/designing-products/lightning-service-providers/
+main_classes: -no-top-padding
+---
+
+<!--
+
+Editor's notes
+
+Illustration sources: https://www.figma.com/file/qr4P17z6WSPADm6oW0cKw2/?node-id=701%3A5911
+
+-->
+
+{% include picture.html
```

Comment:
It's the actual image I had in mind to use, heavily based on this album from the movie drive: https://soundcloud.com/electricyouthmusic/college-and-electric-youth-a-real-hero 😆 

If people don't like it then I'm happy to change it

Reply:
I'm good with the style, but would agree with critique that the image only contains text and could be a better visual representation of the page content.

Reply:
I like the idea of a more creative, playful style, but the typeface looks a little Microsoft Windows 98, IMO. There are other script typefaces that might look nice.

- [Hurricane](https://fonts.google.com/specimen/Hurricane?category=Handwriting&preview.text=Lightning%20Service%20Providers&preview.text_type=custom)
- [Fontjek](https://www.dafont.com/fontjek.font?fpp=200&l[]=10&l[]=1&text=Lightning+Service+Providers)
- [Art Brush](https://www.dafont.com/artbrush.font?fpp=200&l[]=10&l[]=1&text=Lightning+Service+Providers)
- [Arkipelago](https://www.dafont.com/arkipelago.font?fpp=200&l[]=10&l[]=1&text=Lightning+Service+Providers)

At a bare minimum, just not putting most of the image text in all caps would help a lot. If they're going to be all caps, it kind of defeats the purpose of script typeface because a good script typeface is going to have these nice cursive connecting lines.

These are all nits.

Reply:
> but the typeface looks a little Microsoft Windows 98

That's kind of the point 😆 I'll re-do the images if people don't like the aesthetic, these kind of things a pretty subjective though. The font used only has capitalization https://www.wfonts.com/font/mistral.

If you view my source file you can see the mood board I used for this style: https://www.figma.com/file/qr4P17z6WSPADm6oW0cKw2/?node-id=701%3A5911